src/controllers/beneficial.owner.other.controller.ts

```diff
@@ -19,6 +19,8 @@ import { PrincipalAddressKey, PrincipalAddressKeys, ServiceAddressKey, ServiceAd
 import { StartDateKey, StartDateKeys } from "../model/date.model";
 import { v4 as uuidv4 } from 'uuid';
 import { BENEFICIAL_OWNER_OTHER_PAGE, BENEFICIAL_OWNER_TYPE_URL } from "../config";
+import { saveAndContinue } from "../utils/save.and.continue";
+import { Session } from "@companieshouse/node-session-handler";
```

Comment:
Could you please move the import of external resource to the top, It is more readable to have imports aggregated, thanks

Reply:
Updated this.